src/common/ci.utils.js

```diff
@@ -197,3 +198,43 @@ export const destroyNodeImgSize = imgNode => {
   imgNode.removeAttribute("height");
   imgNode.removeAttribute("width");
 };
+
+const createImage = (imageSrc) => {
+  const image = new Image();
+  image.src = imageSrc;
+
+  return image;
+}
+
+export const createIcon = (iconImageSrc, className) => {
+  const icon = document.createElement('div');
+  const iconImage = createImage(iconImageSrc)
+
+  icon.classList.add(className);
+  icon.append(iconImage);
+
+  return icon;
+}
+
+const createModalImageWrapper = (imgSelector, imgNodeSRC) => {
+  const modalImageWrapper = document.createElement('div');
+  const modalImage = new Image()
+
+  modalImage.setAttribute(imgSelector, imgNodeSRC)
```

Comment:
you are mixing between Src and SRC so please check how is to going in the project and you have to type it the same